src/pages/dashboardTab/dashboard.js

```diff
@@ -60,7 +60,7 @@ const Dashboard = ({
                     />
                   </div>
                 </div>
-                <ActiveFiltersQuery />
+                {displayActiveFiltersQuery ? <ActiveFiltersQuery /> : ''}
```

Comment:
This will show up on the search page is there a way we can do it only on the dashboard?